sound/soc/sof/sof-audio.c

```diff
@@ -291,6 +291,18 @@ int sof_route_setup(struct snd_sof_dev *sdev, struct snd_soc_dapm_widget *wsourc
 	sroute->setup = true;
 	return 0;
 }
+EXPORT_SYMBOL(sof_route_setup);
+
+/* helper function to check if the widget belongs to a BE pipeline */
+static bool sof_is_be_pipeline_widget(struct snd_soc_dapm_widget *w)
+{
+	struct snd_sof_widget *swidget = w->dobj.private;
+
+	if (swidget->spipe->be_pipeline)
```

Comment:
I would rather use `be_managed_pipeline`. This flag is not set for example for IPC3 BE pipelines which does not make them non BE pipelines.